torch/_prims/nvfuser_executor.py

```diff
@@ -78,6 +78,17 @@ def _to_nvfuser_constant(arg):
                 return arg
 
         class FusionInterpreter(torch.fx.Interpreter):
+            def run_node(self, node):
+                # Squeeze requires original shape of args[0]
+                # This catches torch.ops.nvprims.squeeze and torch.ops.nvprims.squeeze.default
+                if "squeeze" in str(node.target):
```

Comment:
err, are you sure you want to do a substring match?

Reply:
No, I changed that.